packages/airnode-node/src/config/index.ts

```diff
@@ -56,7 +60,17 @@ export function getEnvValue(envName: string) {
   return process.env[envName];
 }
 
-function validateConfig(supposedConfig: unknown, secrets: Record<string, string | undefined>): Result {
+function validateConfig(
+  supposedConfig: unknown,
+  secrets: Record<string, string | undefined>,
+  shouldValidate = true
```

Comment:
Why one of these has default `true` and the other `false` - it feels inconsistent.

Reply:
Do we actually need the default values?